### PR TITLE
update

### PR DESCRIPTION
**Pull request by t00thp3ck**

### CODE REVIEW

File: aoblipplayer.h [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 #include <QWidget>
 
+#define blip_count 5
```
Comment (oldmud0) [inline aoblipplayer.h]: You should uppercase preprocessor macros; it is good practice.

File: aoapplication.cpp [hunk 2]
```diff
@@ -40,7 +40,8 @@ void AOApplication::construct_lobby()
   int y = (screenGeometry.height()-w_lobby->height()) / 2;
   w_lobby->move(x, y);
 
-  discord->state_lobby();
+  if (get_discord())
```
Comment (oldmud0) [inline aoapplication.cpp]: `discord_enabled()` has better intent.

File: main.cpp [hunk 3]
```diff
@@ -18,7 +18,10 @@ int main(int argc, char *argv[])
 #endif
     AOApplication main_app(argc, argv);
     main_app.construct_lobby();
+
+#ifndef t00thp3ck
```
Comment (oldmud0) [inline main.cpp]: kek

**Comment by t00thp3ck** [comment]
not supposed to remove dependency
supposed to simply allow disabling

**Comment by oldmud0** [comment]
All right. Is there a user interest in this?

**Comment by t00thp3ck** [comment]
i am interested in this
toggle is better than full removal

File: main.cpp (hunk 3 above)
Comment (t00thp3ck) [reply main.cpp]: yes?

File: aoapplication.cpp (hunk 2 above)
Comment (t00thp3ck) [reply aoapplication.cpp]: will change later

File: text_file_functions.cpp [hunk 4]
```diff
@@ -49,7 +49,12 @@ QString AOApplication::read_theme()
   if (result == "")
     return "default";
   else
-    return result;
+      return result;
+}
+
+QString AOApplication::get_ooc_name()
+{
+  return read_config("ooc_name");
```
Comment (oldmud0) [inline text_file_functions.cpp]: Good addition.

**Pull request closed without merging**